website/forms/javascript/events/how-to-use.md

```diff
@@ -6,77 +6,52 @@ title: How to use?
 ### esFormsAfterCaptchaInit
 
 In this example the event is hooked to the `window` object and in event details you have:
-* `esForms` - object.
-* `formId` - not available because this is a global event.
-* `additional` - object from the API response.
 
-```js
-window.addEventListener('esFormsAfterCaptchaInit', ({detail}) => {
-	const {
-		additional: {
-			status,
-		},
-	} = detail;
-
-	if (status === 'success') {
-		const score = data?.response?.score?.toString();
-
-		if (score) {
-			// Do some actions with the score.
-
-			window.dataLayer.push({
-				event: 'recaptcha',
-				recaptchaScore: score,
-			});
-		}
-	}
-});
-```
-
-### esFormsJsFormLoaded
-
-In this example the event is hooked to the `form` element and in event details you have:
-* `esForms` - object.
-* `formId` - form Id this event is a part of.
+- `esForms` - object.
+- `formId` - not available because this is a global event.
+- `additional` - object from the API response.
 
 ```js
 import domReady from '@wordpress/dom-ready';
 
 domReady(() => {
-	const element = document.querySelector('.js-es-block-form');
-
-	if (!element) {
-		return;
-	}
-
-	element?.addEventListener('esFormsJsFormLoaded', ({detail}) => {
+	window.addEventListener('esFormsAfterCaptchaInit', ({ detail }) => {
 		const {
-			formId,
-			esForms,
+			additional: {
+				status,
+			},
 		} = detail;
 
-		// Do some actions with the form.
+		if (status === 'success') {
+			const score = data?.response?.score?.toString();
+
+			if (score) {
+				// Do some actions with the score.
+
+				window.dataLayer.push({
+					event: 'recaptcha',
+					recaptchaScore: score,
+				});
+			}
+		}
 	});
 });
 ```
 
-### esFormsAfterFormSubmitReset
+### esFormsJsFormLoaded
+
+In this example the event is hooked to the `window` element and in event details you have:
 
-In this example the event is hooked to the `form` element and in event details you have:
-* `esForms` - object.
-* `formId` - form Id this event is a part of.
-* `additional` - object from the API response.
+- `esForms` - object.
+- `formId` - form Id this event is a part of.
 
 ```js
-const { store } = window.esForms.store;
+import domReady from '@wordpress/dom-ready';
+
+domReady(() => {
+	window?.addEventListener('esFormsJsFormLoaded', ({ detail }) => {
```

Comment:
why check if `window` object exists if this will only be run in browser where window is always defined?